scripts/eval/yamls/hf_eval.yaml

```diff
@@ -37,9 +37,11 @@ models:
 device_eval_batch_size: 4
 
 # FSDP config for model sharding
-fsdp_config:
-  sharding_strategy: FULL_SHARD
-  mixed_precision: FULL
+# fsdp_config:
+#   sharding_strategy: FULL_SHARD
+#   mixed_precision: FULL
+#   forward_prefetch: True
+#   limit_all_gathers: True
 
-icl_tasks: 'eval/yamls/tasks_light.yaml'
+icl_tasks: 'eval/yamls/tasks.yaml'
```

Comment:
is this change intentional?

Reply:
yeah I want all the configs to reflect the standard gauntlet usage